docs/local_ui_development.md

```diff
@@ -55,4 +55,24 @@ npm install nodemon -g
 Access the UI at https://localhost. For your convenience, local set up uses a TestUserAuthority which accepts any username and password as far as it's a same string.
 To look at a pre-created "athenz" domain during start up, login with "athenz-admin" as username and password.
 
-Now you can keep the UI instance running in the terminal, make changes to the code using your IDE and UI dev server will reload with your changes.
\ No newline at end of file
+Now you can keep the UI instance running in the terminal, make changes to the code using your IDE and UI dev server will reload with your changes.
+
+##Troubleshooting
+###Clean old artifacts
+It is possible for build to fail if Athenz was previously built. To clean old artifacts:
+1. Run `make make remove-all` from the docker directory
+2. Run `docker image ls` to locate images related to Athenz. Remove each of them using `docker image rm <container id>`.
+
+###Manually install self-signed certificate as root on Mac OS
+When you open https://localhost you might get a certificate error as the self-signed certificate isn't trusted.
+To trust the certificate, run the following command:
+`security add-trusted-cert -k %s/Library/Keychains/login.keychain ${BASE_DIR}/docker/sample/CAs/athenz_ca.der\n" "$HOME`
+
+If the command didn't work, you may add it using the UI:
+1. Open Keychain Access app
+2. Select File -> Import items...
+3. Select the certificate located at ${BASE_DIR}/docker/sample/CAs/athenz_ca.der
+4. After the certificate is added to the list, click on it
+5. Expand "Trust"
+6. In the field "When using this certificate:" select "Always Trust".
+7. Reload https://localhost
```

Comment:
Maybe also provide a link to https://github.com/AthenZ/athenz/blob/master/docs/setup_ui.md#ui-access if they don't want to add the self-signed cert to their kaychain?

Reply:
Fixed